File: 3100/golang/3146. Permutation Difference between Two Strings.go
func findPermutationDifference(s string, t string) int {
	res := 0
	for i, cs := range s {
		for j, ct := range t {
			if cs == ct {
				res += abs(i - j)
			}
		}
	}
	return res

}
func abs(x int) int {
	if x < 0 {
		return -x
	}
	return x
}

File: 3100/rust/3146. Permutation Difference between Two Strings.rs
impl Solution {
    pub fn find_permutation_difference(s: String, t: String) -> i32 {
        let mut res=0;
        for (i,cs) in s.char_indices() {
            for (j,ct) in t.char_indices() {
                if cs==ct {
                    res+=(i as i32 - j as i32).abs()
                }
            }
        }
        res
    }
}
